enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -152,7 +154,20 @@ export interface MetabaseComponentTheme {
       };
     };
   };
-}
+
+  collectionBrowser: {
+    breadcrumbs: {
+      expandButton: {
+        backgroundColor: ColorCssVariableOrString;
+        hoverBackgroundColor: ColorCssVariableOrString;
+        textColor: ColorCssVariableOrString;
+        hoverTextColor: ColorCssVariableOrString;
+      };
+    };
+  };
+};
+
+type ColorCssVariableOrString = `var(--mb-color-${ColorName})` | string;
```

Comment:
With this, we get a little bit of auto-complete for finding the right colors for the extended theme

Reply:
oh nice!

Reply:
This doesn't seem to work. Can you double check?
![image](https://github.com/metabase/metabase/assets/1937582/fade93f8-13b0-4002-bfed-319acbb64cac)

Reply:
@WiNloSt  It still seems to be working for me - it's not supposed to prevent other values (since users will also be theming their own things) but it is supposed to give us some level of help when using the IDE

![](https://github.com/metabase/metabase/assets/25306947/b0cfeed7-6aae-4cff-923f-f840b7def69a)

Reply:
VS code doesn't show such autocompletion for me. @npretto Can you help verify if this is another VS Code problem 😆